### PR TITLE
CEO Oscar

### PR DESCRIPTION
**Pull request by d-little**
### Oscar

image

### CODE REVIEW

File: src/server/cards/ceos/Oscar.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          b.br.br;
+          b.opgArrow().chairman();
+        }),
+        description: 'You have +1 influence. Once per game, replace the Chairman with one of your delegates.',
+      },
+    });
+  }
+
+  public override play(player: Player) {
+    const turmoil = player.game.turmoil;
```
Comment (kberg) [inline src/server/cards/ceos/Oscar.ts:26]: Is it possible to discard a CEO? If so, we'll need an onDiscoard.

File: src/server/cards/ceos/Oscar.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      return false;
+    }
+    const turmoil = Turmoil.getTurmoil(player.game);
+    return turmoil.hasDelegatesInReserve(player.id) && turmoil.chairman !== player.id;
+  }
+
+  public action(player: Player): PlayerInput | undefined {
+    const turmoil = Turmoil.getTurmoil(player.game);
+    // This action does not grant TR, so we dont use turmoil.setNewChairman()
+    turmoil.delegateReserve.remove(player.id);
```
Comment (kberg) [inline src/server/cards/ceos/Oscar.ts:42]: Should there be code which moves the chairman to the reserve?

**Review by kberg — COMMENTED** [review]
Nice work.

File: src/server/cards/ceos/Oscar.ts (hunk 2 above)
Comment (d-little) [reply src/server/cards/ceos/Oscar.ts:42]: Yeah.  

Might I suggest adding options to turmoil.setNewChairman(), with a flag for gainTR (default true). That way we keep all of this code in one place?

File: src/server/cards/ceos/Oscar.ts (hunk 1 above)
Comment (d-little) [reply src/server/cards/ceos/Oscar.ts:26]: Lowell discards himself.  Not entirely sure what that relates to here though

File: src/server/cards/ceos/Oscar.ts (hunk 2 above)
Comment (kberg) [reply src/server/cards/ceos/Oscar.ts:42]: Please do. Feel free to do that in this PR or another, your call.

**Comment by d-little** [comment]
Ready for review after the Turmoil turmoil resolved itself

**Review by kberg — APPROVED** [review]
Looks great.

**Pull request merged**